api/config.py

```diff
@@ -61,7 +66,20 @@
 
 def apply_env_variables(config):
     # Overwrite default config values with SCITRAN env variables if available
+
+    # Load auth config from file if available
+    if 'SCITRAN_AUTH_CONFIG_FILE' in os.environ:
+        auth_config = config['auth']
+        file_path = os.environ['SCITRAN_AUTH_CONFIG_FILE']
+        with open(file_path) as config_file:
```

Comment:
Given that, what are your thoughts on `update` below this diff? Do we want this file to be completely authoritative, at which point you can't modify the the config from the DB? Or, leave it as is? Are you comfortable using `update`?